evaluator_test.go

```diff
@@ -231,6 +233,81 @@ func TestClusters(t *testing.T) {
 	}))
 }
 
+func TestClustersBoth(t *testing.T) {
+	testEval(t, NewResult("a", "b", "c"), "clusters(one,three)", multiCluster(map[string]Cluster{
+		"a": Cluster{"CLUSTER": []string{"two", "one"}},
+		"b": Cluster{"CLUSTER": []string{"$ALL"}, "ALL": []string{"one"}},
+		"c": Cluster{"CLUSTER": []string{"three"}},
+	}))
+}
+
+func TestClusterQueryCache(t *testing.T) {
```

Comment:
WDYT about running range spec twice, once with enabled, once with disabled? (Though per above comment I'm not convinced it should be configurable)